---
layout: default
---
# About me

I hold a PhD in Cancer Genetics and I am licensed in Pharmaceutical Sciences. I completed my doctoral studies in King's College London (UK), where I investigated the impact of hydroxyurea treatment on DNA methylation and gene expression in patients with myeloproliferative neoplasms, as well as in a mouse model of the disease ([Contreras Castillo, *et al*., 2021](https://genome.cshlp.org/content/early/2021/07/23/gr.270066.120)).

Currently, I am a postdoc in Katie Sawai's lab (INSERM U1312) at the University of Bordeaux (France). My research focuses on studying the impact of aging on normal hematopoiesis using single-cell multi-omic techniques. 

# Publications
- *G. Jang†, S. Contreras Castillo†, et al.* Stem cell decoupling underlies impaired lymphoid development during aging. Proc. Natl. Acad. Sci. 120, 2017. doi: [10.1073/pnas.2302019120](https://doi.org/10.1073/pnas.2302019120). †denotes equal contribution.
- *S. Contreras Castillo et al.*, “Hydroxycarbamide effects on DNA methylation and gene expression in myeloproliferative neoplasms,” *Genome Research*, vol. 31, no. 8, pp. 1381–1394, Aug. 2021, doi: [10.1101/gr.270066.120](https://doi.org/10.1101/gr.270066.120).
- *S. Contreras Castillo et al.*, “Effect of CYP3A4, CYP3A5, MDR1 and POR Genetic Polymorphisms in Immunosuppressive Treatment in Chilean Kidney Transplanted Patients,” *Frontiers in Pharmacology*, vol. 12, Dec. 2021, doi: [10.3389/fphar.2021.674117](https://doi.org/10.3389/fphar.2021.674117).
- *S. M. Amante et al.*, “Transcription of intragenic CpG islands influences spatiotemporal host gene pre-mRNA processing,” *Nucleic Acids Research*, vol. 48, no. 15, pp. 8349–8359, Sep. 2020, doi: [10.1093/nar/gkaa556](https://doi.org/10.1093/nar/gkaa556).
- *Á. Roco, J. Cayún, S. Contreras, J. Stojanova, and L. Quiñones*, “Can pharmacogenetics explain efficacy and safety of cisplatin pharmacotherapy?,” *Frontiers in Genetics*, vol. 5, no. Nov. Frontiers Media S.A., 2014. doi: [10.3389/fgene.2014.00391](https://doi.org/10.3389/fgene.2014.00391).

# Techniques experience

- Omics 
	+ RNA-seq for low input of RNA (TaKaRa)
	+ Single cell RNA-seq (10x Genomics)
	+ Single cell multiome ATAC + Gene Expression seq (10x Genomics)
	+ Infinium MethylationEPIC (Illumina)

- Flow cytometry
- CRISPR-Cas9 (mESC, primary cells & cell lines)

# Coding experience

- Bulk RNA-seq: fastqc, STAR, Kallisto, sleuth, enrichR
- Single cell RNA/ATAC-seq: cellranger, Seurat, SingleR, SCENIC, GSEA 
- DNA methylation: limma, minfi, RnBeads, wateRmelon, sva, DMRcate
- visualization tools: ggplot2

# Free time

- Tennis
- Street photography, analog & digital 
- Bicycle touring 